service/controller/aws/resource_set.go

```diff
@@ -406,6 +407,10 @@ func getWorkerCount(obj interface{}) (int, error) {
 	return key.WorkerCount(cr), nil
 }
 
+func getWorkerMaxCPUCores(obj interface{}) (int, bool, error) {
+	return 0, false, nil
+}
```

Comment:
We only track CPU cores on KVM. So we could use the provider flag in the resource and then we wouldn't need this.

Reply:
Please ignore comment above. I get why we need this.